data/fvcom.py

```diff
@@ -247,24 +247,6 @@ def weight(r):
 
         return np.squeeze(output)
 
-    def get_path(self, path, depth, time, variable, numpoints=100, times=None, return_depth=False):
```

Comment:
I've deleted `get_path` from fvcom, nemo, and mercator and moved it into model, since the implementations were identical.